assets/js/components/ClusterDetails/ChecksResultOverview.stories.jsx

```diff
@@ -0,0 +1,24 @@
+import { faker } from '@faker-js/faker';
+import ChecksResultOverview from './ChecksResultOverview';
+
+export default {
+  title: 'ChecksResultOverview',
+  component: ChecksResultOverview,
+};
+
+export const Default = {
+  args: {
+    data: {
+      completed_at: faker.date.recent().toISOString(),
+      passing_count: faker.datatype.number(),
+      warning_count: faker.datatype.number(),
+      critical_count: faker.datatype.number(),
+    },
+    error: false,
+    loading: false,
+    onCheckCLick: {},
```

Comment:
`onCheckCLick: () => {}` might fit better.

Reply:
Yeah @nelsonkopliku  makes sence!

@arbulu89  thanks!
oh gosh I fixed it first and brought it back after rebase xD!
Thanks guys!!